docs/appendix/glossary/type.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Type
```

Comment:
```suggestion
title: Type (Resource API)
```
This longer title is only really necessary if we end up wanting to make the configuration/more normal usage of type a separate glossary, like in your pr, & separate that out more! @JessamyT

Reply:
Yeah I'm cool with this just being "type" based on the way you've done this PR!